sources/vega/commands/v1/validator_commands.proto

```diff
@@ -127,3 +127,7 @@ message StateVariableProposal {
   // The state value proposal details
   vega.StateValueProposal proposal = 1;
 }
+
+message ProtocolUpgradeProposal {
+  vega.ProtocolUpgrade proposal = 1;
+}
```

Comment:
Why not just put here directly the fields of the proposal instead of the having another type, which contains them>? Not a big deal, but most other commands does that usually.

Reply:
sure